x/axelarnet/client/cli/tx.go

```diff
@@ -76,21 +74,12 @@ func GetCmdConfirmDeposit() *cobra.Command {
 				return err
 			}
 
-			txID, err := hex.DecodeString(args[0])
-			if err != nil {
-				return err
-			}
-
-			if len(txID) != common.HashLength {
-				return fmt.Errorf("txID should be %d bytes", common.HashLength)
-			}
-
 			burnerAddr, err := sdk.AccAddressFromBech32(args[2])
 			if err != nil {
 				return err
 			}
 
-			msg := types.NewConfirmDepositRequest(cliCtx.GetFromAddress(), txID, args[1], burnerAddr)
+			msg := types.NewConfirmDepositRequest(cliCtx.GetFromAddress(), nil, args[1], burnerAddr)
```

Comment:
update the contructor so not to ask for this parameter, but do not remove the field from the protobuf

Reply:
also, i almost forgot: mark the field as deprecated on the protobuf like this: https://github.com/axelarnetwork/axelar-core/blob/main/proto/axelarnet/v1beta1/tx.proto#L32